common/OpTestSSH.py

```diff
@@ -155,52 +159,131 @@ def get_console(self):
 
         return self.console
 
-    def run_command(self, command, timeout=60):
-        console = self.get_console()
-        if self.prompt:
-            prompt = self.prompt
+    def set_env(self, console):
+        set_env_list = []
+        console.sendline("exec bash --norc --noprofile")
```

Comment:
actually bash is not there in petitboot shell, so this will fail when in skiroot tests. So can you keep it like this.
```
if self.use_default_bash:
            console.sendline("exec bash --norc --noprofile")
```
Failure output:
```
    cmd:0xc1a800010a000000	mode:0x004d04000a000000	stat:0x0a0c0c000a000000
  estat:0x0800fd180a000000	intm:0x000000000a000000	intc:0x00000a2a0a000000
XX [  670.225259089,3] I2C: Transfer error occurred
[  670.225289948,3] I2C: Chip 00000000 Eng. 1 P^Cexec bash --norc --noprofile
PS1=\[console-pexpect\]#
^C
[console-pexpect]#exec bash --norc --noprofile
/bin/sh: exec: line 866: bash: not found
```
After that all tests are failing.
```
  runTest (testcases.PciSlotLocCodes.Skiroot) ... OpTestSystem START STATE: 4 (target 4)
OpTestSystem TRANSITIONED TO: 4
ERROR (30.810s)
  runTest (testcases.PciSlotLocCodes.SkirootDT) ... OpTestSystem START STATE: 4 (target 4)
OpTestSystem TRANSITIONED TO: 4
ERROR (31.120s)
  runTest (testcases.OpTestPCI.PcieLinkErrorsSkiroot) ... OpTestSystem START STATE: 4 (target 4)
OpTestSystem TRANSITIONED TO: 4
ERROR (31.528s)
```